compiler/src/handler/mod.rs

```diff
@@ -43,9 +52,117 @@ use super::compilers::compiler::Compiler;
 /// io::stderr().write_all(&output_data.stderr).unwrap();
 /// 
 pub fn run_compilation(input_data: &InputData) -> Result<OutputData, &'static str> {
-    let compiler = select_compiler(&input_data.compiler_type);
+    
+    let mut updated_input_data = InputData::new(
+        input_data.compiler_type.to_owned(), 
+        input_data.source_code_file_path.to_owned(), 
+        input_data.compiled_directory_path.to_owned(), 
+        input_data.compiler_options.to_owned());
+
+    
+    let raw_options = input_data.compiler_options.clone();
+  
+    // TODO add to logger
+    //println!("Example string: {}", BOTH_OPTIONS_EXAMPLE);
+    
+    let mut config_file_path: PathBuf = env::current_dir().unwrap();
+    config_file_path = config_file_path.join(CONFIG_FILE_NAME);
+    
+    let config = load_config(config_file_path)?;
```

Comment:
are we loading config (from fs) every time when we are processing requests?

Reply:
We now call run_compilation() func from backend. This approach is stateless. So the only way to avoid this is to implement some "compilation and other stuff config" struct to hold all of that data.

Reply:
And such config struct should be stored on the Backend side, @AlexanderSatana, am I correctly understood you? 
We have talked about compiler as a variable stored in Backend and decided to postpone this issue. Maybe it is time to return to this or make separate "config struct", what do you think, @ValentynFk?

Reply:
At the start of the sprint I offered to make some kind of handler struct to make compiler calls stateful, but we agreed that it was not the highest priority at the time. But yeah, it would be great to not load the config every time a request is handled.